src/ResourceManager/Compute/Commands.Compute/Generated/LogAnalytic/LogAnalyticExportRequestRateByIntervalMethod.cs

```diff
@@ -0,0 +1,270 @@
+            var pGroupByThrottlePolicy = (bool?) ParseParameter(invokeMethodInputParameters[3]);
+            parameters.GroupByThrottlePolicy = pGroupByThrottlePolicy;
+            var pBlobContainerSasUri = (string) ParseParameter(invokeMethodInputParameters[4]);
+            parameters.BlobContainerSasUri = pBlobContainerSasUri;
+            var pGroupByResourceName = (bool?) ParseParameter(invokeMethodInputParameters[5]);
+            parameters.GroupByResourceName = pGroupByResourceName;
+            var pToTime = (DateTime) ParseParameter(invokeMethodInputParameters[6]);
+            parameters.ToTime = pToTime;
+            string location = (string)ParseParameter(invokeMethodInputParameters[8]);
+
+            var result = LogAnalyticsClient.ExportRequestRateByInterval(parameters, location);
+            WriteObject(result);
+        }
+    }
+
+    public partial class NewAzureComputeArgumentListCmdlet : ComputeAutomationBaseCmdlet
+    {
+        protected PSArgument[] CreateLogAnalyticExportRequestRateByIntervalParameters()
+        {
+            RequestRateByIntervalInput parameters = new RequestRateByIntervalInput();
+            string location = string.Empty;
+
+            return ConvertFromObjectsToArguments(
+                 new string[] { "Parameters", "Location" },
+                 new object[] { parameters, location });
+        }
+    }
+
+    [Cmdlet(VerbsData.Export, "AzureRmLogAnalyticRequestRateByInterval", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(PSLogAnalyticsOperationResult))]
+    public partial class ExportAzureRmLogAnalyticRequestRateByInterval : ComputeAutomationBaseCmdlet
+    {
+        public override void ExecuteCmdlet()
+        {
+            ExecuteClientAction(() =>
+            {
+                if (ShouldProcess(this.Location, VerbsData.Export))
+                {
+                    var parameters = new RequestRateByIntervalInput();
+                    parameters.FromTime = this.FromTime;
+                    parameters.GroupByOperationName = this.GroupByOperationName;
+                    parameters.IntervalLength = this.IntervalLength;
+                    parameters.GroupByThrottlePolicy = this.GroupByThrottlePolicy;
+                    parameters.BlobContainerSasUri = this.BlobContainerSasUri;
+                    parameters.GroupByResourceName = this.GroupByResourceName;
+                    parameters.ToTime = this.ToTime;
+                    string location = this.Location.Canonicalize();
+
+                    var result = LogAnalyticsClient.ExportRequestRateByInterval(parameters, location);
+                    var psObject = new PSLogAnalyticsOperationResult();
+                    ComputeAutomationAutoMapperProfile.Mapper.Map<LogAnalyticsOperationResult, PSLogAnalyticsOperationResult>(result, psObject);
+                    WriteObject(psObject);
+                }
+            });
+        }
+
+
+        [Parameter(
```

Comment:
Please put the parameters in position order/order that makes the most sense - this determines the order of parameters in the syntax section of the help file.

Reply:
ping